source/lambda/configuration/config_admin.py

```diff
@@ -108,8 +108,7 @@ def __init__(self, logger, context):
         :param context: Lambda context
         """
         self._table_name = self.table_name
-        self._table = boto3.resource("dynamodb").Table(self._table_name)
-        add_retry_methods_to_resource(self._table, ["scan", "get_item", "put_item", "delete_item"], context=context)
+        self._table = DynamoDBUtils.get_dynamodb_table_resource_ref(self._table_name)
```

Comment:
I don't see any reference to setting retries in this file, is standard retry the default now for the dynamo interface? (legacy is the default for clients - [doc](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/retries.html))